dashboards-observability/public/components/synthetics/helpers/test_details_flyout.tsx

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+    EuiButton,
+    EuiComboBox,
+    EuiComboBoxOptionOption,
+    EuiFlexGroup,
+    EuiFlexItem,
+    EuiFlyoutBody,
+    EuiFlyoutFooter,
+    EuiFlyoutHeader,
+    EuiLink,
+    EuiMarkdownFormat,
+    EuiSpacer,
+    EuiText,
+    EuiTitle,
+  } from '@elastic/eui';
+  import { PPL_DOCUMENTATION_URL } from '../../../../common/constants/shared';
+  import _ from 'lodash';
+  import React, { useState } from 'react';
+  import { FlyoutContainers } from '../../common/flyout_containers';
+  import { Group1, Group2, Group3 } from '../../common/helpers/ppl_docs/groups';
+  import { overview } from '../../common/helpers/ppl_docs/overview';
+import { Plt } from '../../visualizations/plotly/plot';
+import { CONTENT_SIZE_FIELD, DOWNLOAD_SPEED_FIELD, DOWNLOAD_TIME_FIELD, REDIRECT_TIMING_FIELD, STARTTRANSFER_TIMING_FIELD, TEST_SUITE_NAME_FIELD, TIMING_FIELDS, URL_FIELD } from '../../../../common/constants/synthetics';
+  
+  type Props = {
```

Comment:
Probably use interface instead of type and name Props to something more specific to this file like TestDetailsFlyoutProps

Reply:
Fixed.